### DISCUSSION TITLE
How to change number box textbox font size

### DISCUSSION THREAD

**Discussion by universal-101** [bdlukaa/fluent_ui#933] [General]
Hi,
I'm trying to implement a series of numberboxes, but unable to set the textbox font size inside numberBox widget(currently the size of the text is too big for my requirements), can anyone please enlighten me.
Thanks a lot.

**Comment by bdlukaa** [comment]
Hello! We currently do not expose the `style` property on NumberBox.

Please, wait for the upcoming version :)
